src/control/server/storage/scm/provider.go

```diff
@@ -562,9 +565,18 @@ func (p *Provider) CheckFormat(req FormatRequest) (*FormatResponse, error) {
 		}
 
 		p.log.Debugf("device %s filesystem: %s", req.Dcpm.Device, fsType)
-		if fsType != fsTypeNone {
-			return res, nil
+
+		switch fsType {
+		case fsTypeExt4:
+			res.Mountable = true
```

Comment:
how come we don't also set res.Formatted = true here?

Reply:
Already set when res was initialized above (default to formatted until proven otherwise).